docs/jax-101/05.1-pytrees.md

```diff
@@ -187,6 +187,22 @@ plt.scatter(xs, forward(params, xs), label='Model prediction')
 plt.legend();
 ```
 
+## Key paths
+
+In a nested pytree, each leaf has a unique "path of keys" corresponding to its place in the tree. JAX allows you to access the key paths and use them when handling a pytree. The corresponding APIs are:
```

Comment:
I think we don't guarantee the path is unique.

```suggestion
In a pytree each leaf has a _key path_. A key path for a leaf is a list of _keys_, where the length of the list is equal to the depth of the leaf in the pytree . Each _key_ is a [hashable object](https://docs.python.org/3/glossary.html#term-hashable) that represents an index into the corresponding pytree node type. The type of the key depends on the pytree node type; for example, the type of keys for `dict`s is different from the type of keys for `tuple`s.

For built-in pytree node types, the set of keys for any pytree node instance is unique. For a pytree comprising nodes with this property, the key path for each leaf is unique.

The APIs for working with key paths are:
```

Reply:
Done.